### PR TITLE
Add note that minio is necessary for tests on MacOS.

### PR DESCRIPTION
**Pull request by markbader**
### Description:
- Adds a line to the CONTRIBUTING.md file for installation of the test setup on MacOS.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -140,6 +140,11 @@ npm install --global proxay
 # or if you're using yarn
 yarn global add proxay
 ```
+When running the `./test.sh` script on iOS, it is also necessary to install `minio` using the following command:
```
Comment (hotzenklotz) [inline CONTRIBUTING.md:143]:
```suggestion
When running the `./test.sh` script on MacOS, it is also necessary to install `minio` using the following command:
```

Comment (hotzenklotz) [reply CONTRIBUTING.md:143]: iOS? That is the mobile phone OS from Apple. Do you mean MacOS, their desktop operating system?

Comment (hotzenklotz) [reply CONTRIBUTING.md]: And why is this an OS specific requirement? `minio` is an S3-API compatible object storage service. I believe we use that for testing S3 compatibility locally. Hence, you probably need that on Linux/(Windows) too if you want to run the tests, right?

**Comment by hotzenklotz** [comment]
And why is this an OS specific requirement? minio is an S3-API compatible object storage service. I believe we use that for testing S3 compatibility locally. Hence, you probably need that on Linux/(Windows) too if you want to run the tests, right?

See https://vscode.dev/github/scalableminds/webknossos-libs/blob/more-docstrings/webknossos/tests/constants.py#L37

**Comment by normanrz** [comment]
On Linux, we use docker for minio. On macOS, we use the locally installed minio. https://github.com/scalableminds/webknossos-libs/blob/master/webknossos/tests/constants.py#L33-L51

**Pull request merged**